### ISSUE TITLE
onliner

### ISSUE THREAD

**Issue by rmx152** [xzerios/Discord-Auto-Advertiser#8]
onliner aint working rn anyone know how to fix? I got custom status and normal status working but it isnt keeping it online. Also as a suggestion you should add multi token support w seperate settings based on token

**Comment by xathail** [comment]
Most of the code is due an update which is why the onliner isn't working most likely. The multitoken idea sounds pretty good.
I'll probably update this some point in the next week.

**Comment by rmx152** [comment]
Ah ok I see, I tried to fix myself and implement custom rpc but couldn't get it to work, some of the features I added which I think would probably be good were : Status to see if its currently running, and option to stop, storing channel ids under server ids so that there is a custom delay based on server id (useful for servers with cooldowns), skip sending message if most recent message is the same (prevent clutter), periodacally keyword search based on server id (useful for a few servers) (to get this to work though its almost mandatory to store channel ids under server ids, otherwise it will overwrite everything, removed dm deleter (couldnt really see its use), and made auto dm responder only reply to each user in a session once by appending their id to a list. Trying to do multi acc now and see if i can fix onliner and add custom rpc, but not sure how to implement code from other projects, such as online-forever, into it. Hope some of the features are good enough to implement, your code is a good base to work off.
